src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: AvenirBook;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

Comment:
check out point 7 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)